controller/admin-controller.js

```diff
@@ -313,6 +315,68 @@ const downloadPaymentReport = async (req, res, next) => {
   }
 };
 
+//For the admin to be able to add events for the user
+const getAllEvents = async (req, res) =>{
+  const events = await Event.find({event_id: req.event.id});
```

Comment:
if you are getting all event .. you are not suppose to pass anything inside of find method